packages/store/package.json

```diff
@@ -3,38 +3,37 @@
   "license": "MIT",
   "version": "1.41.0",
   "description": "Store",
-  "types": "./types/ethers-contracts/",
```

Comment:
I am unclear on the side effects of this, can no longer remember why I removed it 🙈

Reply:
reverted for now to see if everything still builds, etc.